common/mixnet-contract/src/error.rs

```diff
@@ -0,0 +1,9 @@
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq)]
+pub enum MixnetContractError {
+    #[error("Overflow Error")]
+    OverflowError(#[from] cosmwasm_std::OverflowError),
+    #[error("reward_blockstamp field not set, set_reward_blockstamp must be called before attempting to issue rewards")]
```

Comment:
It does not seem that `BlockstampNotSet` variant is being used anywhere

Reply:
Actually the same is true for `OverflowError` and entire `MixnetContractError` enum